zerver/lib/user_mutes.py

```diff
@@ -14,7 +15,7 @@ def get_user_mutes(user_profile: UserProfile) -> List[Dict[str, int]]:
     return [
         {
             "id": row["muted_user_id"],
-            "timestamp": datetime_to_timestamp(row["date_muted"]),
+            "timestamp": datetime_to_timestamp(assert_is_not_none(row["date_muted"])),
```

Comment:
Hmm, why is it possible for `date_muted` to be None?  It's not declared as optional in `MutedUser`.

Reply:
Hmm, that contradicts with the stubs. I guess it might be related to `django-stubs`' implementation for `.values`

Reply:
Hmm, yeah, maybe `.values` doesn't know enough about the model structure to determine whether fields might be None?  Not sure, could be worth a mini chat.zulip.org thread.